src/models/Automaton.java

```diff
@@ -269,18 +281,11 @@ public void makeInputEnabled() {
                     Edge newEdge = new Edge(loc, loc, input, true, resCDD.getGuard(getClocks()), new ArrayList<>());
                     getEdges().add(newEdge);
                 }
-
             }
         }
 
-    }
-
-    public void addTargetInvariantToEdges() {
-        for (Edge edge : getEdges()) {
-            CDD targetCDD = edge.getTarget().getInvariantCDD();
-            CDD past = targetCDD.transitionBack(edge);
-            if (!past.equiv(CDD.cddTrue()))
-                edge.setGuard(past.conjunction(edge.getGuardCDD()).getGuard(getClocks()));
+        if (initialisedCdd) {
+            CDD.done();
         }
     }
 }
```

Comment:
I see that Github is 'complaining' that this file does not have a new line character at the end 😆 .